docs/contracts/visibility-and-getters.rst

```diff
@@ -148,7 +148,7 @@ If you have a ``public`` state variable of array type, then you can only retriev
 single elements of the array via the generated getter function. This mechanism
 exists to avoid high gas costs when returning an entire array. You can use
 arguments to specify which individual element to return, for example
-``data(0)``. If you want to return an entire array in one call, then you need
+``data[0]``. If you want to return an entire array in one call, then you need
```

Comment:
This does not refer to the example above as-is. It's referring to a hypothetical situation where `data` is an array rather than a single `uint`.

We could use the name from the example below instead to avoid the ambiguity:

```suggestion
``myArray(0)``. If you want to return an entire array in one call, then you need
```

Reply:
Thanks a lot, updated as suggested.